src/ui/public/react_components.js

```diff
@@ -0,0 +1,15 @@
+import 'ngreact';
```

Comment:
IMHO there would be no benefit in implementing but not exporting some components. We won't be able to consistently apply the "currently being used" criterion for long anyway - it'll probably degrade to a mish-mash after a while.